src/components/pagination/table-pagination.tsx

```diff
@@ -58,13 +70,21 @@ export const TablePagination = ({
   onChangeRowsPerPage,
   onChangePage,
   className,
+  selectStyles,
+  locales,
```

Comment:
just use default 
```
  locales = {
    itemsPerPage: 'Items per page',
    displayingItems: (rangeStart, rangeEnd) => `Displaying ${rangeStart}-${rangeEnd} of ${count} items`,
    currentPage: (currentPage: number, pagesAmount: number) => `${currentPage + 1} of ${pagesAmount} pages`,
  },
```

Reply:
So should 'locales' be mandotory passed?

Reply:
no, we can avoid passing `locales`